### ISSUE TITLE
Update 3.5.0/3.5.1 fails on migrations

### ISSUE THREAD

**Issue by Ferrrrrre** [barrelstrength/craft-sprout-forms#363]
### Description (screenshots, logs, ...)
When updating the plugin from 3.4.3 to 3.5.1 on one of our site, it keeps failing on the migrations.

```
Database Exception: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'sproutforms_integrations' already exists
The SQL being executed was: CREATE TABLE sproutforms_integrations (
id int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
formId int(11) NOT NULL,
name varchar(255) NOT NULL,
type varchar(255) NOT NULL,
settings text,
enabled tinyint(1) DEFAULT FALSE,
dateCreated datetime(0) NOT NULL,
dateUpdated datetime(0) NOT NULL,
uid char(36) NOT NULL DEFAULT '0'
) ENGINE=InnoDb DEFAULT CHARSET=utf8

Migration: barrelstrength\sproutforms\migrations\m181120_000000_add_integrations_table

Output:

> create table {{%sproutforms_integrations}} ...Exception: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'sproutforms_integrations' already exists
The SQL being executed was: CREATE TABLE sproutforms_integrations (
id int(11) NOT NULL AUTO_INCREMENT PRIMARY KEY,
formId int(11) NOT NULL,
name varchar(255) NOT NULL,
type varchar(255) NOT NULL,
settings text,
enabled tinyint(1) DEFAULT FALSE,
dateCreated datetime(0) NOT NULL,
dateUpdated datetime(0) NOT NULL,
uid char(36) NOT NULL DEFAULT '0'
) ENGINE=InnoDb DEFAULT CHARSET=utf8 (/Users/ferre/Code/site.technopolis/vendor/yiisoft/yii2/db/Schema.php:664)
```

If we delete all the sprout forms tables, we get yet another error about the foreign key of that table. 

What are we doing wrong?

### Additional info

- Craft version: Craft Pro 3.3.14
- PHP version: 7.2.19
- Database driver & version:  MySQL 5.7.24

**Comment by BenParizek** [comment]
Thanks @Ferrrrrre 

We're looking into it and will update this thread once we have a fix.

**Comment by BenParizek** [comment]
@Ferrrrrre This should be resolved in Sprout Forms v3.6.4
